machines/config_machines.xml

```diff
@@ -84,6 +83,9 @@ Author: Alex Domingo (Vrije Universiteit Brussel)
         <command name="load">CESM-deps/2-intel-2019b</command>
       </modules>
     </module_system>
+    <resource_limits>
+      <resource name="RLIMIT_STACK">-1</resource>
```

Comment:
is this something useful in general?

Reply:
yes, I'll open a ticket about this, we might want to have an overall setting